core-container/entry.sh

```diff
@@ -229,10 +229,8 @@ function run_init() {
 function run_mailbox_controller() {
     echo "--< Starting mailbox-controller >--"
     wait-kubestellar-ready
-    # TODO: remove the kcp dependency in PR3
     get_kcp_kubeconfig
-    KUBECONFIG=$kcp_kubeconfig
-    kubectl ws root:espw
+    KUBECONFIG=$SM_CONFIG
```

Comment:
(A) Don't you need an `export KUBECONFIG` so that this setting gets to the mailbox-controller process?

(B) Assigning to this global variable make this whole script very fragile. I would either

```
function run_mailbox_controller() (
...
)
```

or

```
function run_mailbox_controller() {
...
    if ! KUBECONFIG=$SM_CONFIG mailbox-controller -v=${VERBOSITY} ; then
...
```